### DISCUSSION TITLE
TSS / turret plugin for IDEs

### DISCUSSION THREAD

**Discussion by hanseartic** [kalepail/stellar-quest-bounties#121] [Bounty Ideas]
# Manage turrets/smart contacts from within the IDE

I would like to see a plugin for intelliJ or VS Code so that people could just write their code and then upload to a turret from within the IDE.

To big of a scope for a bounty?

**Comment by ElliotFriend** [comment]
That could be really useful! I can't speak to scope, I've no idea how much work it is to create ide plugins. But, I think it's certainly something worth writing up
